adback-solution-to-adblock/public/class-ad-back-public.php

```diff
@@ -78,13 +78,19 @@ public function enqueueScripts()
         $me = $this->getMyInfo();
         $mess = $this->getCacheMessages();
 
-        if(is_array($me)) {
-            if(!empty($me['analytics_domain']) && !empty($me['analytics_script'])) {
+        if (is_array($me)) {
+            if (!empty($me['analytics_domain']) && !empty($me['analytics_script'])) {
                 echo "<script>(function (a,d){var s,t;s=d.createElement('script');s.src=a;s.async=1;t=d.getElementsByTagName('script')[0];t.parentNode.insertBefore(s,t);})('https://".$me['analytics_domain']."/".$me['analytics_script'].".js', document);</script>";
             }
 
-            if(!empty($me['message_domain']) && !empty($me['message_script'])) {
-                if($mess->display) {
+            if (!empty($me['message_domain']) && !empty($me['message_script'])) {
+                if (
+                    $mess->display
+                    && (
+                        !current_user_can('manage_options')
+                        || (current_user_can('manage_options') && get_option('adback_admin_show_message'))
```

Comment:
I think is "not the get_option" because on admin, the checkbox is not to show the message